Client/src/components/common/Table/index.js

```diff
@@ -18,9 +19,10 @@ const Table = ({ headings, rows, history }) => {
       </thead>
       <tbody>
         {rows.map((row, index) => {
+          const rowContents = row.slice(1);
           return (
-            <tr className='table-row' key={index} onClick={() => history.push(row.id)}>
-              {row.map((rowcontent, index) =>
+            <tr className='table-row' key={index} onClick={() => history.push(`${pathname}/${row[0]}`)}>
```

Comment:
@Fatmasiam Another alternative to ```onClick={() => history.push(`${pathname}/${row[0]}`)}``` is to wrap this in all in a ```<Link to={`${pathname}/${row[0]}`}>```
It's not an important change to make (and you might have to change some css as a result) but it's an idea for next time.  Then you don't have to put the inline function definition (the `() => {}`) in your render, which would make this component use less memory.